src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{

```

Comment:
There should not be a redundant empty line after the class declaration. Please remove the empty line to improve code readability, adhering to the checklist item regarding redundant empty lines.